src/Compilers/CSharp/Portable/CSharpParseOptions.cs

```diff
@@ -86,6 +86,7 @@ public override IEnumerable<string> PreprocessorSymbolNames
             kind: other.Kind,
             preprocessorSymbols: other.PreprocessorSymbols)
```

Comment:
It is used by "With" methods that clone and modify one specific part with construction initializer.
For instance, `return new CSharpParseOptions(this) { Kind = kind };`